R_program/Main.R

```diff
@@ -6,6 +6,7 @@
 library(DBI)
 library(RSQLite)
 library(rSOILWAT2)
+library(stringr)
 
```

Comment:
@kpalmqui I added stringr to separate the group name from the parameter name, for example
```
row.names(pctlive) <- str_match(pctlive[ , 1], "(.*)_pctlive")[ , 2]
```
I could, however, add another column so instead of having
```
sagebrush_biomass | value | ...
```
We would have 
```
| sagebrush | biomass | value | ...
```
in the .csv file. That way we wouldn't need the str_match function and therefore we wouldn't need stringr.